From a5e0e304d30ed7aa1eacc6cca04514913b8d2385 Mon Sep 17 00:00:00 2001
From: Mikkel Oscar Lyderik Larsen <m@moscar.net>
Date: Thu, 8 Dec 2016 23:33:21 +0100
Subject: [PATCH] Add missing SOURCELINK_SUFFIX

Adds DOCUMENTATION_OPTIONS.SOURCELINK_SUFFIX which is expected by the
searchtools.js

Related sphinx commit:
https://github.com/sphinx-doc/sphinx/commit/71dd8bfbf94417ad55b2444e1dbd219db266f335
---
 sphinx_rtd_theme/layout.html     | 3 ++-
 sphinx_rtd_theme/layout_old.html | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/sphinx_rtd_theme/layout.html b/sphinx_rtd_theme/layout.html
index e8d9b5f..db5828f 100644
--- a/sphinx_rtd_theme/layout.html
+++ b/sphinx_rtd_theme/layout.html
@@ -170,7 +170,8 @@
             VERSION:'{{ release|e }}',
             COLLAPSE_INDEX:false,
             FILE_SUFFIX:'{{ '' if no_search_suffix else file_suffix }}',
-            HAS_SOURCE:  {{ has_source|lower }}
+            HAS_SOURCE:  {{ has_source|lower }},
+            SOURCELINK_SUFFIX: '{{ sourcelink_suffix }}'
         };
     </script>
     {%- for scriptfile in script_files %}
diff --git a/sphinx_rtd_theme/layout_old.html b/sphinx_rtd_theme/layout_old.html
index deb8df2..f0dc2af 100644
--- a/sphinx_rtd_theme/layout_old.html
+++ b/sphinx_rtd_theme/layout_old.html
@@ -91,7 +91,8 @@
         VERSION:     '{{ release|e }}',
         COLLAPSE_INDEX: false,
         FILE_SUFFIX: '{{ '' if no_search_suffix else file_suffix }}',
-        HAS_SOURCE:  {{ has_source|lower }}
+        HAS_SOURCE:  {{ has_source|lower }},
+        SOURCELINK_SUFFIX: '{{ sourcelink_suffix }}'
       };
     </script>
     {%- for scriptfile in script_files %}
